lib/roku/index.js

```diff
@@ -0,0 +1,53 @@
+var assert = require('assert');
+var https = require('../https');
+
+var apiUrl = {
+	production: 'https://apipub.roku.com/listen/transaction-service.svc/validate-transaction'
+};
+
+exports.verifyPayment = function (payment, cb) {
+	try {
+		assert.equal(typeof payment.devToken, 'string', 'Developer ID must be a string');
+		assert.equal(typeof payment.receipt, 'string', 'Receipt must be a string');
+		assert.equal(payment.receipt.match(/\w/g).length, 32, 'Receipt must contain 32 digits');
+		assert.equal(payment.receipt.match(/-/g).length, 4, 'Receipt must contain 4 dashes');
+	} catch (error) {
+		return process.nextTick(function () {
+			cb(error);
+		});
+	}
+
+	var requestUrl = apiUrl.production + '/' + payment.devToken + '/' + payment.receipt;
+
+	https.get(requestUrl, {headers: {Accept: 'application/json'}}, function (error, res, resultString) {
+		if (error) {
+			return cb(error);
+		}
+
+		if (res.statusCode !== 200) {
+			return cb(new Error('Received ' + res.statusCode + ' status code with body: ' + resultString));
+		}
+
+		var resultObject = null;
+
+		try {
+			resultObject = JSON.parse(resultString);
+		} catch (error) {
+			return cb(error);
+		}
+
+		if (resultObject.errorMessage) {
+			return cb(new Error(resultObject.errorMessage));
+		}
+
+		resultObject.expirationDate = new Date(parseInt(resultObject.expirationDate.substr(6))).getTime();
```

Comment:
@ronkorving Roku sends back the following data when validating a receipt:
```
{
  "errorCode": null,
  "errorDetails": null,
  "errorMessage": "",
  "status": 0,
  "amount": 4.99,
  "cancelled": false,
  "channelId": abc,
  "channelName": "Motortrend OnDemand",
  "couponCode": null,
  "currency": "usd",
  "expirationDate": "/Date(1490755828000-0700)/",
  "originalPurchaseDate": "/Date(1483242628000-0800)/",
  "partnerReferenceId": null,
  "productId": "5KAZUPGB.0RF0",
  "productName": "BASIC - US MONTHLY",
  "purchaseDate": "/Date(1483242628000-0800)/",
  "quantity": 1,
  "rokuCustomerId": "e833bebfb72a57b1a7e579904bd8b0af",
  "tax": 0,
  "total": 0,
  "transactionId": "abc"
}
```

You'll notice that any dates field include more than just the numerical value: `/Date(1483242628000-0800)/`. 

To extract the date, we used the following suggestion: http://stackoverflow.com/a/4511910/2395590

This helps us extract the date and return it back formatted as milliseconds.